.github/actions/squidex-sync/action.yml

```diff
@@ -26,14 +29,40 @@ runs:
       uses: ./.github/actions/setup-environment
       with:
         environment-name: ${{ inputs.environment-name }}
-    - name: Update Squidex schema
+    - name: Update Squidex schema branch
       shell: bash
+      if: ${{ inputs.environment-name=='Branch' }}
       run: |
         sq config add $SQUIDEX_APP_NAME $SQUIDEX_CLIENT_ID $SQUIDEX_CLIENT_SECRET -u $SQUIDEX_BASE_URL
         sq config use $SQUIDEX_APP_NAME
         .github/scripts/squidex-sync/config-rules.sh
-        sq sync in packages/squidex/schema/crn -t schemas -t rules --delete
+
+        if [[
+          "$APP" == "crn-server" && "$CRN_API_URL" == *"asap"*
```

Comment:
what do we need this for?

Reply:
We don't. I'm testing it works on the Branch environment.